_literature_review_2022/2021_Amanda_Weekends.md

```diff
@@ -0,0 +1,64 @@
+bibtex: |-
+  @article{doi:10.1177/1354856520913865,
+        author = {Amanda C Cote and Brandon C Harris},
+        title ={‘Weekends became something other people did’: Understanding and intervening in the habitus of video game crunch},
+        journal = {Convergence},
+        volume = {27},
+        number = {1},
+        pages = {161-176},
+        year = {2021},
+        doi = {10.1177/1354856520913865},
+
+        URL = { 
+                https://doi.org/10.1177/1354856520913865
+            
+        },
+        eprint = { 
+                https://doi.org/10.1177/1354856520913865
+            
+        }
+        ,
+            abstract = { ‘Crunch’ – a period of unpaid overtime meant to speed up lagging projects – is a common labor practice in the video game industry and persists despite many costs to developers. To understand why, we conducted a critical discourse analysis of Game Developer magazine (2000–2010) to explore how industry members perceive and discuss gamework (1) in a publication for developers, by developers and (2) during the first decade in which serious conversations about labor emerge in the games industry. Our analysis found that many gameworkers treat crunch as ‘inevitable’ due to three specific themes: games as an unmanageable creative industry, an anti-corporate ethos, and a stereotypical developer identity based on passion and perfectionism. These constructions – combined with the industry’s project-based nature and cultures of passion and secrecy – build crunch into the habitus of gamework, helping reproduce exploitative labor practices. However, habitus can and does change over time, providing interested employees, companies, and labor organizers a means to intercede in existing work practices. We suggest a multipronged intervention that could build a healthier, more sustainable habitus of gamework. }
```

Comment:
remove the abstract. In general, it's okay to have it in the bibtex, but in this case it will affect the layout of the page a bit.